xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -141,6 +141,27 @@
       <code>require(['jquery', 'xwiki-meta'], function ($, xm) {
   'use strict';
   $(document).ready(function() {
+
+    /** Add autocomplete attributes for accessibility purpose. */
```

Comment:
Minor remark but I don't think autocomplete it's an accessibility feature only: for me it's mainly something handy to avoid retyping stuff. At least there's no mention of accessibility when I read the doc in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete.

Reply:
Now I feel a bit strange to have that feature only at User profile edition: it means that when you register you don't have autocomplete features filling the names / emails and you have it when you edit the profile. It might feel a bit inconsistent.

Reply:
Thanks Simon for your feedback. You're right, these attributes are useful also for avoiding retyping, I've updated the comment. As for registration: indeed, we need to update the form as well, I have created a distinct issue about that earlier today: [XWIKI-18837](https://jira.xwiki.org/browse/XWIKI-18837). We have also one for the login screen username / password fields: [XWIKI-18811](https://jira.xwiki.org/browse/XWIKI-18811).